packages/zowe-explorer/README.md

```diff
@@ -83,7 +85,23 @@ Before using the base profile functionality, ensure that you have [Zowe CLI](htt
 
 For more information, see [Integrating with API Mediation Layer](https://docs.zowe.org/stable/user-guide/cli-usingcli.html#integrating-with-api-mediation-layer).
 
-### Sample use cases
+#### Log in to the Authentication Service
+
+Use the Log in to the **Authentication Service** feature to regenerate a new token for your base profile.
+
+**Follow these steps:**
+
+1. Open Zowe Explorer.
+2. Right-click your profile.
+3. Select the **Log in to Authentication Service** option.
+
+   You will be prompted to enter your username and password beforehand.
+
+The token is stored in the default base profile .yaml file.
+
+<!-- TODO For more information, see [](link to the Zowe Docs page). -->
```

Comment:
Should we also briefly mention Log out here?

Reply:
Done